zerver/tests/fixtures/slack_message_conversion.json

```diff
@@ -84,6 +99,11 @@
       "name": "italic_and_strike_conversion",
       "input": "_~italic~_ and ~_strike_~",
       "conversion_output": "*~~italic~~* and ~~*strike*~~"
+    },
+    {
+      "name": "unsupported_bold_test",
+      "input": "*abc*\n*def*\n*ghi*\n*jkl*\n\n*ABC ABC*",
+      "conversion_output": "**abc**\n*def*\n**ghi**\n*jkl*\n\n**ABC ABC**"
```

Comment:
The reformatter couldn't support converting this bold string because the SLACK_BOLD_REGEX have a capture group that matches for a character before asterisk "*"

SLACK_BOLD_REGEX = r"""
                    (\n|^|[ -(]|[+-/]|[:-?]|\{|\[|\_|\||\^|~)
                    ...

In this case, both `*abc*` and `*def*` shares one character(`\n`) in between them, so `*def*` won't get matched because that character was used to match `*abc*`.

There's probably a way to support this case, but I don't think the potential complexity would be worth it because in practice Slack export shouldn't produce this kind of string, if two words got bolded separately like this example, then when exported it will just turn up like this: *foo bar*.